modules/secure_storage_connector/outputs.tf

```
@@ -0,0 +1,11 @@
+output "bucket_name" {
+  value = module.file_storage.bucket_name
+}
+
+output "bucket_arn" {
+  value = module.file_storage.bucket_arn
+}
```

I've starting perforing this approach, but up to you!
```suggestion
output "bucket" {
  value = module.file_storage
}
```

---

done